### PR TITLE
Infra Integration Playground

### PR DESCRIPTION
**Pull request by l-monninger**
# Summary
- **Status**: `draft` 
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.

# Changelog

# Testing

# Outstanding issues

### CODE REVIEW

**Comment by l-monninger** [comment]
gali testing

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/26

**Comment by mzabaluev** [comment]
Nit: the permissions on workflow files should not be changed to 0755, as they are not executable.

File: protocol-units/settlement/mcr/contracts/broadcast/DeployMCR.s.sol/21433/run-1716384062.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (mzabaluev) [inline protocol-units/settlement/mcr/contracts/broadcast/DeployMCR.s.sol/21433/run-1716384062.json:1]: This looks like some deployment output added by mistake, please clean up.

File: protocol-units/settlement/mcr/contracts/out/Base.sol/CommonBase.json [hunk 2]
```diff
@@ -0,0 +1 @@
...,"forge-std/=lib/forge-std/src/","murky/=lib/murky/","openzeppelin-contracts/=lib/openzeppelin-contracts/","openzeppelin/=lib/openzeppelin-contracts/"],"optimizer":{"enabled":true,"runs":200},"metadata":{"bytecodeHash":"ipfs"},"compilationTarget":{"lib/forge-std/src/Base.sol":"CommonBase"},"evmVersion":"paris","libraries":{}},"sources":{"lib/forge-std/src/Base.sol":{"keccak256":"0x4ff1a785311017d1eedb1b4737956fa383067ad34eb439abfec1d989754dde1c","urls":["bzz-raw://f553622969b9fdb930246704a4c10dfaee6b1a4468c142fa7eb9dc292a438224","dweb:/ipfs/QmcxqHnqdQsMVtgsfH9VNLmZ3g7GhgNagfq7yvNCDcCHFK"],"license":"MIT"},"lib/forge-std/src/StdStorage.sol":{"keccak256":"0x651d84d948832f0ef45686417aa68ffb871378fa788a4123dbf37844903c66f3","urls":["bzz-raw://ff20f5ac9de3dc3ba86b1bf1f2723174e82ce3499ad67cb0ccfa7d28baeee678","dweb:/ipfs/QmRZkUFKz7AmF7yk6o317sk822HHhGVPXZQgX8G4LfYfft"],"license":"MIT"},"lib/forge-std/src/Vm.sol":{"keccak256":"0x30d73c4fea34a791ad7334dd701df60fbb565c643556aadd6621e49e48895d1f","urls":["bzz-raw://be9cf80e55eff0f49ad0ab3c44214e56a3e53b5ecafe311a521f5472fa4bbf59","dweb:/ipfs/QmQHrZZoscvkfKSD2m6jhktrh8ieSLV2fg9DHGDHJiBkjT"],"license":"MIT OR Apache-2.0"}},"version":1},"id":0}
```
Comment (mzabaluev) [inline protocol-units/settlement/mcr/contracts/out/Base.sol/CommonBase.json:1]: This likewise, and below.

**Comment by l-monninger** [comment]
@mzabaluev I will talk through tomorrow. I should have marked this as draft.

File: protocol-units/settlement/mcr/contracts/out/Base.sol/CommonBase.json (hunk 2 above)
Comment (l-monninger) [reply protocol-units/settlement/mcr/contracts/out/Base.sol/CommonBase.json:1]: Yeah, these can be ignored.

**Pull request closed without merging**